components/Projects/FiltersBar/FiltersBar.tsx

```diff
@@ -85,7 +85,7 @@ const FiltersBar: React.FC = () => {
                 <>
                   <div
                     ref={filterRef}
-                    className="z-[101] absolute -bottom-[23.5rem] md:-bottom-[14rem] right-2 px-[34px] py-[27px] rounded-md border border-blue-600 min-w-[400px] md:min-w-0 md:w-[863px] h-[360px] md:h-[209px] p-5 bg-gray-50 dark:bg-gray-500"
+                    className="z-[101] absolute -bottom-[23.5rem] md:-bottom-[14rem] right-2 px-[34px] py-[27px] rounded-md border border-blue-600 min-w-[400px] md:min-w-0 md:w-[863px] h-[360px] md:h-[209px] p-5 bg-gray-50 dark:bg-gray-700"
```

Comment:
Great catch! but in Figma it suppose to be `bg-gray-600`
![image](https://github.com/Maakaf/maakaf-website/assets/12512909/c7571f34-3dbb-44ef-b4a1-e10b525109ba)